ts/private/ts_project_options_validator.js

```diff
@@ -131,25 +131,6 @@ function main(_a) {
             )
         }
     }
-    if (options.noEmit) {
-        console.error(
-            'ERROR: ts_project rule ' +
-                target +
-                " cannot be built because the 'noEmit' option is specified in the tsconfig."
-        )
-        console.error(
-            'This is not compatible with ts_project, which always produces outputs.'
-        )
-        console.error(
-            '- If you mean to only typecheck the code, use the tsc_test rule instead.' +
-                '  (see https://github.com/aspect-build/rules_ts/tree/main/examples/typecheck_only)'
```

Comment:
I don't see an update to that example in this PR?

Reply:
Updated.

Note that I had to change the example to be a *passing* type-check and not failing, now that we can't use `expected_exit_code = 2`.

Reply:
yeah that's understood thx